src/datacatalogtordf/document.py

```diff
@@ -0,0 +1,141 @@
+"""Document module for mapping a document to rdf.
+
+This module contains methods for mapping a document object to rdf
+according to the
+`dcat-ap-no v.2 standard <https://doc.difi.no/review/dcat-ap-no/#klasse-distribusjon>`__
+
+Example:
+    >>> from datacatalogtordf import Document
+    >>> # Create a document:
+    >>> document = Document()
+    >>> document.identifier = "http://example.com/documents/1"
+    >>> document.title = {"en": "The Python Language Reference Manual"}
+"""
+from __future__ import annotations
+
+from typing import Optional
+
+from rdflib import BNode, DCTERMS, FOAF, Graph, Literal, Namespace, RDF, URIRef
+
+from datacatalogtordf.uri import URI
+
+DCAT = Namespace("http://www.w3.org/ns/dcat#")
+
+
+class Document:
+    """A class representing a foaf:Document.
+
+    Attributes:
+        identifier (URI): A URI uniquely identifying the document
+        title (dict): A title given to the document. key is langauge code
+        language (str): The organzation's identifier
```

Comment:
Her er vel ikkje beskrivelsen heilt korrekt? Altså: `language` er vel ikkje "the organization's identifier"?

Reply:
Oj, ja dette er jo helt feil. Takk for påpekning!